src/commonMain/kotlin/exchange.dydx.abacus/processor/markets/MarketsSummaryProcessor.kt

```diff
@@ -24,6 +28,70 @@ internal class MarketsSummaryProcessor(parser: ParserProtocol, calculateSparklin
         return modify(existing, markets)
     }
 
+    internal fun testSubscribed(content: Map<String, Any>): InternalStatePerpetualMarkets {
+        val payload = parser.asNativeMap(content["markets"])
```

Comment:
With Indexer codegen already landed, we can use static types for input payload with something like this now:

`        val marketResponse = Json.decodeFromString<IndexerPerpetualMarketResponse?>(content.toJson())`